README.md

```diff
@@ -73,25 +73,19 @@ After you're finished please remove all the comments and instructions!
 > Describe the tech stack and include only the relevant sections that apply to your project.
 
 <details>
-  <summary>Client</summary>
+  <summary>HTML</summary>
   <ul>
     <li><a href="https://reactjs.org/">React.js</a></li>
   </ul>
 </details>
 
 <details>
-  <summary>Server</summary>
```

Comment:
- [ ] Kindly use [the Microverse Readme template](https://github.com/microverseinc/readme-template) and make sure all the main sections present in yours and placeholders are replaced with your project and personal data. The only optional section is a FAQ. Please remove the template logo on the top, if you have one, you can replace it. This is crucial to have a professional and instructive repository. 👌